Resources/Prototypes/Entities/Markers/Spawners/Random/laws.yml

```diff
@@ -0,0 +1,27 @@
+- type: entity
+  name: Law Board Random Spawner
```

Comment:
```suggestion
  name: law Board Random Spawner
```

Reply:
Most of the other spawners use upper case throughout. Not sure why I'd make the first word lower..?
![image](https://github.com/user-attachments/assets/6f5a669b-6551-4842-b251-d8c25adb6650)
![image](https://github.com/user-attachments/assets/3c5e6c13-9c35-4e97-970c-4ccb1bd446db)